client/disabled/client.go

```diff
@@ -0,0 +1,29 @@
+package disabled
+
+import (
+	"context"
+
+	"github.com/multiversx/mx-chain-core-go/data/sovereign"
+)
+
+type client struct{}
+
+// NewClient creates a new instance of disabled client
+func NewClient() *client {
+	return &client{}
+}
+
+// Send -
```

Comment:
This is not a mock, you should add a comment here + below saying that it does nothing or returns empty objects